[ADD] events: add documentation (#702)
odoo/documentation-user: events/overview/track_talks.rst

======================
Track and Manage Talks
======================

> Allow partners to submit talk proposals so they can share their knowledge and expertise, and to add
> extra value to your event.

Reviewer: ```suggestion
Allow partners to submit talk proposals so they can share their knowledge and expertise. Good speakers and talks also add extra value to your event.
```

> speakers and talks also add extra value to your event.

Reviewer 2: Good speakers and talks are what makes your event great.

Configuration
=============

> | Go to :menuselection:`Events --> Configuration --> Settings`, and enable *Schedule & Tracks*.

Reviewer: ```suggestion
| Go to :menuselection:`Events --> Configuration --> Settings` and enable *Schedule & Tracks*.
```

> | Now, in your event's form, once *Website menu* is enabled, the options *Tracks on Website* and
>   *Track Proposals on Website* become available. Enabling them add the menus *Talks*,
>   *Talk Proposals*, and *Agenda* to your website.

Reviewer: ```suggestion
| Once *Website menu* is enabled, the options *Tracks on Website* and
  *Track Proposals on Website* become available. Enabling them adds the menus *Talks*,
  *Talk Proposals*, and *Agenda* to your website.
```

.. image:: media/track_proposals_website.png
   :align: center
   :alt: View of an event form and the options to track and propose speaker in Odoo Events

> From there, users see a list of the tasks and of the whole agenda, besides having the ability to
> propose their own talks.

Reviewer: ```suggestion
Besides having the ability to suggest their own talks, the website now allows users to see a list of the whole agenda and scheduled talks.
```

.. image:: media/website_menus.png
   :align: center
   :alt: View of the published website and the menus talks, talk proposal and agenda in Odoo Events

.. tip::
   To handle tracks internally and not have the menus *Talks*, *Talk Proposals*, and *Agenda*
   online, use the menu *Tracks* individually.

Publishing speaker proposals
----------------------------

Once partners have filled in the appropriate form and submitted their proposals, a new *Proposal* is

> instantly created under the menu *Track* in your event form.

Reviewer: ```suggestion
instantly created under the menu *Track*.
```

.. image:: media/proposals_list.png
   :align: center
   :alt: View of the talks’ proposals page emphasizing the column proposal in Odoo Events

If the proposal is accepted and can go online, simply open its form and click on *Go to Website*.
From the website page, turn *Published* on.

.. image:: media/publish_speakers.png
   :align: center

>    :alt: View of the website page to publish a proposed talk for Odoo Events

Reviewer: I think you should zoom on the "unpublished".
